### PR TITLE
Localized strings for the dotnet-pack command.

### PR DESCRIPTION
**Pull request by vsccarl**
Added localization for the pack command. All localizable strings from the command are now placed in a new file, LocalizableStrings.cs.

### CODE REVIEW

**Comment by piotrpMSFT** [comment]
@dotnet-bot test OSX x64 Release Build please.

**Comment by piotrpMSFT** [comment]
@MattGertz

File: src/dotnet/commands/dotnet-pack/PackCommand.cs [hunk 1]
```diff
@@ -17,37 +17,37 @@ public static int Run(string[] args)
...
+                Description = LocalizableStrings.AppDescription,
                 AllowArgumentSeparator = true,
                 ArgumentSeparatorHelpText = HelpMessageStrings.MSBuildAdditionalArgsHelpText
             };
 
             cmd.HelpOption("-h|--help");
 
-            var output = cmd.Option("-o|--output <OUTPUT_DIR>",
-                "Directory in which to place outputs",
+            var output = cmd.Option($"-o|--output <{LocalizableStrings.CmdOutputDir}>",
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-pack/PackCommand.cs]: NIT: I would prefer that when you put parameters in a new line that each of them have their own new line. Line this:

cmd.Option(
    $"...",
    Localiza...,
    CommandOptionType...);

Comment (livarcocc) [inline src/dotnet/commands/dotnet-pack/PackCommand.cs]: Same in other locations here.

File: src/dotnet/commands/dotnet-pack/PackCommand.cs [hunk 2]
```diff
@@ -17,37 +17,41 @@ public static int Run(string[] args)
...
                 CommandOptionType.SingleValue);
-            var versionSuffix = cmd.Option("--version-suffix <VERSION_SUFFIX>",
-                "Defines what `*` should be replaced with in version field in project.json",
+            var versionSuffix = cmd.Option(
+                $"--version-suffix <{LocalizableStrings.CmdVersionSuffix}>",
+                LocalizableStrings.CmdVersionSuffixDescription,
                 CommandOptionType.SingleValue);
-            var serviceable = cmd.Option("-s|--serviceable", 
-                "Set the serviceable flag in the package", 
+            var serviceable = cmd.Option("-s|--serviceable",
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-pack/PackCommand.cs]: You moved some of the parameters to their own lines, but you left this one and a couple others. "-s|--serviceable" should also be in its own line. Do a pass and see other cases that may apply.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test OpenSUSE13.2 x64 Debug Build please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test RHEL7.2 x64 Release Build please.

**Pull request merged**